server/parsec/components/account.py

```diff
@@ -97,6 +97,33 @@ class VaultItems:
     items: dict[HashDigest, bytes]
 
 
+VALIDATION_CODE_VALIDITY_DURATION_SECONDS: int = 3600
+"""
+How much seconds a validation code remain usable (valid)
+"""
+
+VALIDATION_CODE_MAX_FAILED_ATTEMPS: int = 3
+"""How much an user can "guess" the code"""
+
+
+@dataclass
+class ValidationCodeInfo:
+    code: ValidationCode
+    created_at: DateTime
+    failed_attemps: int = 0
+
+    def is_expired(self, now: DateTime) -> bool:
+        """Check that the validation code is not expired"""
+        return self.created_at.add(seconds=VALIDATION_CODE_VALIDITY_DURATION_SECONDS) >= now
```

Comment:
The result is the same, but I would find `(now - self.created_at) > VALIDATION_CODE_VALIDITY_DURATION_SECONDS` more readable